doc/API/centreon-api-v2.yaml

```diff
@@ -1947,7 +2003,28 @@ components:
         hosts:
           type: object
           $ref: '#/components/schemas/Monitoring.Host_min'
-
+    Configuration.Proxy:
+      type: object
+      required:
+        - url
+        - port
+        - user
+        - password
+      properties:
+        url:
+          type: string
+          description: Url of the proxy
+        port:
+          type: integer
+          description: Port of the proxy
+          minimum: 0
```

Comment:
I think minimum is one

Reply:
A port is coded in 16 bits = 65536 ports (from 0 to 65535)

Reply:
but technically, is it possible to expose port 0 ?